airflow/utils/log/es_task_handler.py

```diff
@@ -262,3 +267,24 @@ def close(self):
         super().close()
 
         self.closed = True
+
+    def log_name(self):
```

Comment:
```suggestion
    @property
    def log_name(self):
```

Reply:
Done, looks better, thanks. In the process pylint had some complaint about overriden properties, I turned it into an abstract property.